.vsts-ci.yml

```diff
@@ -0,0 +1,84 @@
+resources:
+- repo: self
+  clean: true
+queue:
+  name: VSEng-MicroBuildVS2017
+  demands: Cmd
+variables:
+  BuildConfiguration: Release
+  TeamName: Roslyn-Project-System
```

Comment:
NIT: We are the CLI and SDK team.

Reply:
That's right but  your drops are still being uploaded to` \\cpvsbuild\drops\Roslyn\Roslyn-Project-System\cli-migrate [Windows][x64][master]`. 
It will be easy to change once we switch the build definition to YAML.